### PR TITLE
Improve codestyle of traverse.py module and its tests

### PR DESCRIPTION
**Pull request by ghost**
I've refactored some code to make it more compatible with `pycodestyle`. I used `pylama` to do this.

### CODE REVIEW

File: sympy/strategies/tests/test_traverse.py [hunk 1]
```diff
@@ -1,65 +1,80 @@
...
 
 def test_expr_fns():
-    from sympy.strategies.rl import rebuild
-    from sympy import Add
-    x, y = map(Symbol, 'xy')
     expr = x + y**3
-    e = bottom_up(lambda x: x + 1, expr_fns)(expr)
-    b = bottom_up(lambda x: Basic.__new__(Add, x, 1), basic_fns)(expr)
+    e = bottom_up(lambda _: _ + 1, expr_fns)(expr)
+    b = bottom_up(lambda _: Basic.__new__(Add, _, 1), basic_fns)(expr)
```
Comment (moorepants) [inline sympy/strategies/tests/test_traverse.py]: These changes make the code less readable than it was. Please revert these two lines.

File: sympy/strategies/tests/test_traverse.py [hunk 2]
```diff
@@ -1,65 +1,80 @@
-from sympy.strategies.traverse import (top_down, bottom_up, sall, top_down_once,
-        bottom_up_once, basic_fns)
+from sympy.strategies.traverse import (
+    top_down,
+    bottom_up,
+    sall,
+    top_down_once,
+    bottom_up_once,
+    basic_fns
+)
```
Comment (moorepants) [inline sympy/strategies/tests/test_traverse.py]: The previous version is how the majority of the SymPy codebase deals with this kind of import. In general, you should follow the conventions that already exist in the codebase.

**Comment by moorepants** [comment]
Please address my two comments and I'm +1 for merging. Thanks.

File: sympy/strategies/tests/test_traverse.py (hunk 1 above)
Comment (flamyowl) [inline sympy/strategies/tests/test_traverse.py]: Value of `x` from outer scope is being overshadowed here. Can be the the lambda argument be renamed for let's say `v` or you would insist on `x`?

**Comment by flamyowl** [comment]
Done, with one exception - I've renamed the lambdas parameters from `x` to `v`, as `x` from outer scope was overshadowed here. Can it stay that way?

**Pull request merged**